mw-webapp/src/component/sidebar/SidebarContent/SidebarContent.module.scss

```diff
@@ -0,0 +1,78 @@
+.dialogOverlay {
+  background-color: rgba(0, 0, 0, 0.9);
+  position: fixed;
+  inset: 0;
+  animation: overlayShow 350ms cubic-bezier(0.16, 1, 0.3, 1);
+}
+
```

Comment:
Add to vars colors, timers and etc please (all file)

Reply:
Done